app/ui-utils/client/lib/ReactionList.js

```diff
@@ -0,0 +1,26 @@
+import './ReactionList.html';
+
+import { HTML } from 'meteor/htmljs';
+import { Template } from 'meteor/templating';
+
+
+import { createTemplateForComponent } from '../../../../client/reactAdapters';
+
+// createTemplateForComponent('reactionListContent', () => import('./reactionListContent'));
+
+Template.ReactionList.helpers({
+	reactionData() {
+		const instance = Template.instance();
+
+		return instance.data;
+	},
+});
+
+createTemplateForComponent(
+	'ReactionListContent',
+	() => import('./reactionListContent'),
+	{
+		// eslint-disable-next-line new-cap
+		renderContainerView: () => HTML.DIV({ class: 'reaction-list' }),
+	},
+);
```

Comment:
As `createTemplateForComponent` already creates a template which renders the component *while passing the template data as component props*, I don't think you need the `ReactionList` wrapper.